cfgen/cfgen

```diff
@@ -41,10 +43,10 @@ class DiskRef():
 
 __version__ = '0.14'
 
-faultToleranceInfo = NamedTuple('faultToleranceInfo', [
-                            ('max_drives_per_node', int),
-                            ('max_drives_per_ctrl', int),
-                            ('min_drives_per_node', int)])
+faultToleranceInfo = NamedTuple('faultToleranceInfo',
```

Comment:
Why was this change needed? kindly avoid the unrelated changes.

Reply:
This was falke8 error indentation existing in this statement. Code remains the same.